### DISCUSSION TITLE
cannot save db configuration (using docker compose installation)

### DISCUSSION THREAD

**Discussion by bmccann36** [Budibase/budibase#2485] [Help] [answered]
I am brand new to budibase.
Followed the instructions here https://docs.budibase.com/getting-started#i-just-want-to-run-and-manage-budibase-myself-using-docker
### expected behavior
I enter in my db connection details and they are persisted

### actual behavior
In the network tab I see no API request is made, which is strange because previously I did see requests to the backend when I clicked save. However even when there is a request to the backend, it sends the default placeholder values of the form and not the values I have input.
In other words the host is always localhost. A save success message shows up after hitting save but my hostname or other connection info is never actually saved and when attempting a connection it will always attempt with the default inputs (localhost:5432/postgres)

I also tried the installation using npm cli and had the same issue.
I also tried the digital ocean droplet which froze up every time I tried to create a new database. 

I don't see anything unusual in the docker compose logs, could I be missing some configuration?

**Comment by mjashanks** [comment]
That is strange. I setup postgres connections all the time, and dont experience this.

What Budibase version are you running? You can find this in the Updates section.

Also... What browser and OS are you running?

**Reply by bmccann36** [reply]
budibase version: 0.9.116
OS: big sur 11.5.2
Browser: chrome

I try putting a host into this field, and hit save
<img width="903" alt="Screen Shot 2021-08-27 at 9 14 12 AM" src="https://user-images.githubusercontent.com/28324264/131133081-fc10a0ad-63e6-4024-a5a8-4ed184a0f036.png">

This is always the request to the api (according to dev tools network tab)
<img width="931" alt="Screen Shot 2021-08-27 at 9 14 33 AM" src="https://user-images.githubusercontent.com/28324264/131133191-944f7aca-8ae4-43e6-a082-740081c77d11.png">

none of the inputs, including the ssl setting seem to have any effect

thanks

## update

when I switch to firefox the UI does seem to pass the correct values to the back end in the PUT request
<img width="492" alt="Screen Shot 2021-08-27 at 9 26 17 AM" src="https://user-images.githubusercontent.com/28324264/131134684-8a4c0a85-2427-422f-9826-ef1c66678d8f.png">

however after successful save my saved DB config goes back to displaying the default values. I would imagine it should instead show me the current values of my saved configuration?
<img width="915" alt="Screen Shot 2021-08-27 at 8 15 31 AM" src="https://user-images.githubusercontent.com/28324264/131134935-334d17ba-e98d-4811-a2a8-7c7da8ff2fd5.png">
after saving my db config (to connect to a remote database hosted in aws) it continues to try to connect to localhost

**Reply by kevmodrome** [reply]
Hi Brian!

This was just fixed yesterday and will go out with the next release (coming very soon)

**Reply by bmccann36** [reply]
thanks @kevmodrome + @mjashanks 
in the meantime can you recommend a previous stable release that I can play around with?

**Reply by mjashanks** [reply]
Hey Brian. You could use `develop`, which has the fix now.

https://hub.docker.com/layers/budibase/apps/develop/images/sha256-95da46732654da90cb391bb90d07d3e172ea6b10a280c64af63c83b69dd285c4?context=explore

**Comment by berma83** [comment, marked as answer]
Hi @bmccann36,

I have a "solution" for this.

Export the app and manually edit the database config:

`"config":{"host":"IP","port":5432,"database":"DBNAME","user":"USER","password":"PASSWORD","ssl":false}`

Works for me.

**Reply by bmccann36** [reply]
@berma83 this works thanks

**Comment by oswaldB** [comment]
Hello all, 

Any progress on the new release?

Thank you
